### PR TITLE
Add 4 new authors to authors.yml

### PR DESCRIPTION
**Pull request by havi**
TODO: 
- [ ] Add John's avatar (done but pixilated)
- [x] Add Alex's avatar
- [x] Add Olabayo avatar
- [x] Add Lukonde avatar

Current images lifted from Twitter and LinkedIn, need to confirm these are correct:

### CODE REVIEW

File: _source/_data/authors.yml [hunk 1]
```diff
@@ -802,3 +802,33 @@ alina-yurenko:
   web: https://dev.to/alina_yurenko
   twitter: https://twitter.com/alina_yurenko
   bio: I ❤️ all things tech, open source and community. Developer 🥑 for @GraalVM at @OracleLabs, helping @jhipster, ex @googledevgroups
+  
+alex-doukas:
+  full_name: "Alex Doukas"
+  display_name: "Alex Doukas"
+  avatar:
```
Comment (mraible) [inline _source/_data/authors.yml]: Missing avatar.

File: _source/_data/authors.yml [hunk 2]
```diff
@@ -802,3 +802,33 @@ alina-yurenko:
   web: https://dev.to/alina_yurenko
   twitter: https://twitter.com/alina_yurenko
   bio: I ❤️ all things tech, open source and community. Developer 🥑 for @GraalVM at @OracleLabs, helping @jhipster, ex @googledevgroups
+  
+alex-doukas:
+  full_name: "Alex Doukas"
```
Comment (mraible) [inline _source/_data/authors.yml]: Quotes aren't needed around these values. If you look at the others, they work just fine without quotes.

File: _source/_data/authors.yml (hunk 1 above)
Comment (havi) [reply _source/_data/authors.yml]: yup. that's why i saved as draft. waiting for avatar.

File: _source/_data/authors.yml (hunk 2 above)
Comment (havi) [inline _source/_data/authors.yml]:
```suggestion
  full_name: Alex Doukas
```

File: _source/_data/authors.yml [hunk 3]
```diff
@@ -802,3 +802,33 @@ alina-yurenko:
...
+  display_name: "Alex Doukas"
+  avatar:  
+  linkedin: https://www.linkedin.com/in/alexdoukas/
+  bio: I'm Alex, a front end developer. My main area of expertise is web development and everything that comes along with it. I also have extensive knowledge of topics such as software testing, UX design, big data, social media marketing, and SEO techniques.
+  
+john-gramila:
+  full_name: "John Gramila"
```
Comment (havi) [inline _source/_data/authors.yml]:
```suggestion
  full_name: John Gramila
```

File: _source/_data/authors.yml [hunk 4]
```diff
@@ -802,3 +802,33 @@ alina-yurenko:
...
+  display_name: "John Gramila"
```
Comment (havi) [inline _source/_data/authors.yml]:
```suggestion
  display_name: John Gramila
```

File: _source/_data/authors.yml [hunk 5]
```diff
@@ -802,3 +802,33 @@ alina-yurenko:
...
+  avatar: avatar-JGramila.jpg
+  web: https://toughpixels.com/
+  twitter: https://twitter.com/jgramila
+  bio: John is a developer and writer working in Chicago. Failure and success both create growth, and growth never stops.
+
+lukonde-mwila:
+  full_name: "Lukonde Mwila"
```
Comment (havi) [inline _source/_data/authors.yml]:
```suggestion
  full_name: Lukonde Mwila
```

File: _source/_data/authors.yml [hunk 6]
```diff
@@ -802,3 +802,33 @@ alina-yurenko:
...
+  display_name: "Lukonde Mwila"
```
Comment (havi) [inline _source/_data/authors.yml]:
```suggestion
  display_name: Lukonde Mwila
```

File: _source/_data/authors.yml [hunk 7]
```diff
@@ -802,3 +802,33 @@ alina-yurenko:
...
+  avatar: avatar-LMwila.jpg
+  web: https://toughpixels.com/
+  twitter: https://twitter.com/LuKE9ine
+  bio: Lukonde is a Principal Technical Evangelist at SUSE and is an AWS Container Hero. He specializes in cloud and DevOps engineering and cloud-native technologies. He is passionate about sharing knowledge through various mediums and engaging with the developer community at large.
+
+
```
Comment (havi) [inline _source/_data/authors.yml:835]:
```suggestion
olabayo-balogun:
  full_name: Olabayo Balogun
  display_name: Olabayo Balogun
  avatar: avatar-OBalogun.jpg
  linkedin: https://www.linkedin.com/in/olabayobalogun/
  bio: "A .Net, Blazor and Angular software engineer who is committed to building technological infrastructures capable of solving the challenges of netizens now and in the future. "
  

```
one more bio

File: _source/_data/authors.yml [hunk 8]
```diff
@@ -802,3 +802,40 @@ alina-yurenko:
...
+  avatar: avatar-OBalogun.jpg
+  linkedin: https://www.linkedin.com/in/olabayobalogun/
+  bio: "A .Net, Blazor and Angular software engineer who is committed to building technological infrastructures capable of solving the challenges of netizens now and in the future. "
+  
+
+  
+  
+  
+  
+
```
Comment (bdemers) [inline _source/_data/authors.yml]:
```suggestion
  bio: "A .Net, Blazor and Angular software engineer who is committed to building technological infrastructures capable of solving the challenges of netizens now and in the future. "

```

File: _source/_data/authors.yml [hunk 9]
```diff
@@ -802,3 +802,34 @@ alina-yurenko:
   web: https://dev.to/alina_yurenko
   twitter: https://twitter.com/alina_yurenko
   bio: I ❤️ all things tech, open source and community. Developer 🥑 for @GraalVM at @OracleLabs, helping @jhipster, ex @googledevgroups
+  
+alex-doukas:
+  full_name: Alex Doukas
+  display_name: "Alex Doukas"
+  avatar:
```
Comment (bdemers) [inline _source/_data/authors.yml]:
```suggestion
  avatar:  avatar-ADoukas.jpg
```

**Review by bdemers — APPROVED** [review]

**Pull request merged**